src/test/resources/Hotels.features/Hotels.feature

```diff
@@ -1,54 +1,56 @@
 Feature: Hotels.com
```

Comment:
At line 1 you can have some tag like @regression or @hotels. These tags will help to run all scenarios of this page together using TestRunner class.